src/pages/index.js

```diff
@@ -4,7 +4,10 @@ import PopupWithForm from "../components/PopupWithForm.js";
 import PopUpWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/Userinfo.js";
 import Section from "../components/Section.js";
-import "../pages/index.css"
+import "../pages/index.css";
+import Api from "../components/Api.js";
+import PopUpWithForm from "../components/PopupWithForm.js";
+import PopUpWithConfirm from "../components/PopUpWithConfirm.js";
 
 const initialCards = [
```

Comment:
From the 8th sprint all constants (`initialCards` and `settings`) should be in file `constants.js` in folder `utils` to clean up the main file.